test/README.md

```diff
@@ -123,36 +123,36 @@ with the istio/istio code base.  Both the istio/istio repo and the istio/cni rep
 that the CNI works properly.
 
 In order to run the e2e tests in a local environment confirm:
-
 1. That you can successfully run the desired Istio e2e tests in your local environment without the CNI enabled
 
 2. That your local environment supports the requirements for the CNI plugin
 
-To run the Istio e2e test first, clone the Istio repo in your local environment.  Then, there are two options to run the tests.
+To run any of the Istio e2e test follow these steps:
 
-1. Run the Istio make target  
-```console
-make e2e_simple_cni
-```
-This runs the `e2e_simple` with the latest nightly CNI image pushed to gcr.io/istio-release.  The HUB and TAG value can be overridden using the environement variables:
+1. Clone the Istio repo in your local environment.
+
+2. Install the CNI components as decribed on the main README or other sources. This might include building your own images and/or charts or pulling the images and charts from official repositories.
```

Comment:
Add a link for "the main README" and "other sources"

Reply:
You should be able to link back to the "Usage" heading like this:

```
Install the CNI components as described in the [Usage section](#Usage).
```

Reply:
I can improve this,  but two points.  As above the best link is probably to istio side docs and they aren't stable yet so this is problematic.   Additionally, our longer term plan is to introduce a an option directly in the e2e tests to install the CNI as part of the test.   So I was a bit lazy here figuring this would also change in the matter of a few weeks.

Reply:
I would just put the reference link back to the Usage section and not add the other doc link here.